docs/Plugins/webhook.md

```diff
@@ -12,6 +12,12 @@ When you create an Incoming Webhook within DevLake, DevLake generates a unique U
 
 In v0.14+, users can push "incidents" and "deployments" required by DORA metrics to DevLake via Incoming Webhooks.
 
+Webhooks are meant to be used at the lowest level that you want to relate incidents with deployments. For example, if you want to relate incidents at the individual service level, you will need a webhook per service. If you wish to relate incidents at the product level, you will need a webhook for the product. This is because incidents that are reported to the webhook will be related to the last deployment that was reported to the webhook with a timestamp that is before the incident's timestamp.
+
+Diagram of the relationship between incidents and deployments:
+
+![Change Failure Reporting](/img/Plugins/change-failure.png)
```

Comment:
Hi, you don't need to add a new image file. You can re-use this picture referenced in the CFR doc https://devlake.apache.org/docs/Metrics/CFR#how-is-it-calculated.